.gitmodules

```diff
@@ -3,4 +3,4 @@
 	url = git@github.com:econia-labs/inbox.git
 [submodule "src/typescript/frontend/public/static"]
 	path = src/typescript/frontend/public/static
-	url = https://github.com/tradingview/charting_library.git
+	url = git@github.com:tradingview/charting_library.git
```

Comment:
Is this relying on the private or public version?

If public, how about we keep it as `https` for accessibility?

(In multiple cases we've had downstream devs who aren't aware of SSH auth on GitHub and flag submodule issues)

Reply:
That's the private version, and yes I agree, I'll change them both to `https`, is there any reason we have `inbox` as ssh?

Reply:
So if it is the private version then unfortunately it'll have to use SSH

But we can use https for public repos

Please do go ahead and update it for inbox then